lib/puppet/util/ldap/manager.rb

```diff
@@ -44,7 +44,8 @@ def create(name, attributes)
   # Open, yield, and close the connection.  Cannot be left
   # open, at this point.
   def connect
-    raise ArgumentError, "You must pass a block to #connect" unless block_given?
+    #TRANSLATORS '#connect' is a method name and and should not be translated, 'block' refers to a Ruby code block
```

Comment:
does this mean that `block` should not be translated, or are you just giving context for how `block` should be translated?

Reply:
I was giving context for the translation